lib/src/main/java/fi/epicbot/toster/context/ConfigContext.kt

```diff
@@ -83,4 +83,11 @@ class ConfigContext {
     fun setSwipeOffset(offsetPx: Int, offsetFactor: Double) {
         config.swipeOffset = SwipeOffset(offsetPx, offsetFactor)
     }
+
+    /**
+     * If you want to disable demo mode for tests use this function.
+     */
+    fun enableDemoMode(enable: Boolean = true) {
```

Comment:
you already have it as default value
```suggestion
    fun enableDemoMode(enable: Boolean) {
```